homeassistant/config_entries.py

```diff
@@ -149,6 +149,7 @@ class ExampleConfigFlow(config_entries.ConfigFlow):
     'sonos',
     'tradfri',
     'zone',
+    'upnp',
```

Comment:
that's not how they teach the alphabet 😉

Reply:
Already resolved by @balloob.